beacon-chain/core/helpers/committee.go

```diff
@@ -295,3 +295,67 @@ func Shuffling(
 	// Split the shuffled list into epoch_length * committees_per_slot pieces.
 	return utils.SplitIndices(shuffledIndices, committeesPerEpoch), nil
 }
+
+// AttestationParticipants returns the attesting participants indices.
+//
+// Spec pseudocode definition:
+//   def get_attestation_participants(state: BeaconState,
+//     attestation_data: AttestationData,
+//     bitfield: bytes) -> List[ValidatorIndex]:
+//     """
+//     Returns the participant indices at for the ``attestation_data`` and ``bitfield``.
+//     """
+//     # Find the committee in the list with the desired shard
+//     crosslink_committees = get_crosslink_committees_at_slot(state, attestation_data.slot)
+//
+//	   assert attestation_data.shard in [shard for _, shard in crosslink_committees]
+//     crosslink_committee = [committee for committee,
+//     		shard in crosslink_committees if shard == attestation_data.shard][0]
+//
+//	   assert verify_bitfield(bitfield, len(crosslink_committee))
+//
+//     # Find the participating attesters in the committee
+//     participants = []
+//     for i, validator_index in enumerate(crosslink_committee):
+//         aggregation_bit = get_bitfield_bit(bitfield, i)
+//         if aggregation_bit == 0b1:
+//            participants.append(validator_index)
+//    return participants
+func AttestationParticipants(
+	state *pb.BeaconState,
+	attestationData *pb.AttestationData,
+	bitfield []byte) ([]uint64, error) {
+
+	// Find the relevant committee.
+	crosslinkCommittees, err := CrosslinkCommitteesAtSlot(state, attestationData.Slot, false)
```

Comment:
Nitpick for the future, but for readability it would be good to have a small comment /* registry change */ in the last arg of a the crosslinks func for readers to know what that bool stands for. No need to do here, just something to keep in mind for later.